app/build.gradle

```diff
@@ -10,6 +10,8 @@ android {
         targetSdkVersion rootProject.ext.targetSdkVersion
         versionCode 1
         versionName "1.0"
+
+        setProperty("archivesBaseName", "ZoomLayout_v${versionName}_(${versionCode})")
```

Comment:
What will this do?

Reply:
When building an apk (release or debug) the resulting apk file will be called this way.
I have too much `app-debug.apk` files in my life already :stuck_out_tongue: